ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2486,6 +2486,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-long-duration
     workflow: cucushift-installer-rehearse-gcp-ipi-private
+- as: gcp-ipi-proxy-etcd-encryption-f3
```

Comment:
@jianlinliu Does the frequency looks good to you ?

Reply:
Hmm, any special reason here? @xiaojiey

Reply:
The Security Profiles Operator releases separately. I would like to have a pipeline to trigger a full test for this operator on all supported versions to save time for the release.

Reply:
Ideally, we should set a proper test frequency to meet Security Profiles Operator releases schedule. For example, if its release cycle is once 3 weeks, I think f14 should be okay, if once 2 weeks, then f7 should be enough. So that we can balance the test results and cloud cost.

Reply:
Thanks for the explanation. The current release frequency is about every 6 weeks. I will set it to f28 instead.